frame/assets/src/lib.rs

```diff
@@ -564,8 +564,7 @@ pub mod pallet {
 			let who = T::Lookup::lookup(who)?;
 
 			let f = DebitFlags { keep_alive: false, best_effort: true };
-			let burned = Self::do_burn(id, &who, amount, Some(origin), f)?;
-			Self::deposit_event(Event::Burned(id, who, burned));
```

Comment:
`do_burn` emits this event, shouldn't need it twice:
https://github.com/paritytech/substrate/blob/66c85aeb29fbf4786e76be328cf73241f72c2f18/frame/assets/src/functions.rs#L345